spec/e2e/features/short_form/filling_sale_application.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Short Form Application
+    As a web user
+    I should be able to fill out the short form application
+    In order to apply online to a sale listing
+
+    Scenario: Submitting a basic application, creating an account on the confirmation page
+        Given I go to the first page of the "Sale Test Listing" application
+        When I hit the Next button "1" time
+        And I wait "5" seconds
```

Comment:
Did you add this in because without the wait it fails? have we figured out any other way to wait for a page to load?

Reply:
I put it when I was trying to make it work. Now it works without it as well, so will remove it.

Reply:
cool. hopefully it won't end up being super flaky 😕

Reply:
Removed. yeah, I would hate to see it as well